app/src/main/java/org/wikipedia/analytics/eventplatform/EditAttemptStepEvent.kt

```diff
@@ -42,10 +44,13 @@ class EditAttemptStepEvent(private val event: EditAttemptStepInteractionEvent) :
         private fun submitEditAttemptEvent(action: String, editorInterface: String, pageTitle: PageTitle) {
             EventPlatformClient.submit(EditAttemptStepEvent(EditAttemptStepInteractionEvent(action,
                 WikipediaApp.instance.appInstallID, "", editorInterface,
-                INTEGRATION_ID, "", WikipediaApp.instance.getString(R.string.device_type).lowercase(), 0,
-                    if (AccountUtil.isLoggedIn) AccountUtil.getUserIdForLanguage(pageTitle.wikiSite.languageCode) else 0,
```

Comment:
This is _literally_ the only place where the UserID is used - for analytics purposes, in a single analytics funnel. Can there be a better way of getting the UserID, possibly without any additional network calls at all?